garage/tf/core/parameterLayer.py

```diff
@@ -0,0 +1,54 @@
+"""Parameter layer in TensorFlow."""
+import tensorflow as tf
+from tensorflow.keras.layers import Layer as KerasLayer
+from tensorflow.python.ops.gen_array_ops import broadcast_to
+
+
+class ParameterLayer(KerasLayer):
+    """
+    Parameter layer based on tf.keras.layers.Layer.
+
+    Used as layer that could be broadcast to a certain shape to
+    match with input variable during training.
+    Example: A trainable parameter variable with shape (2,), it needs to be
+    broadcasted to (32, 2) when applied to a batch with size 32.
+    """
+
+    def __init__(self,
+                 length,
+                 scope='ParameterLayer',
+                 initializer='ones',
```

Comment:
Here the default value of initializer is a string, but in [here](https://github.com/rlworkgroup/garage/pull/481/files#diff-f27d474d4aed442478e7e44fdc2de671R148) you passed a function, will this be a problem?

Reply:
`tf.keras` accepts predefined strings and callable, so I think should be fine.